plugins/fileservices/fileservices.cpp

```diff
@@ -2147,7 +2147,7 @@ FILESERVICES_API char *  FILESERVICES_CALL fsfResolveHostName(const char *hostna
 
 static void checkExternalFileRights(ICodeContext *ctx, CDfsLogicalFileName &lfn, bool rd,bool wr)
 {
-    StringAttr extpath;
+    StringAttr extpath(lfn.get());
```

Comment:
Do we need to introduce this var?
Could we use lfn.get() directlty in lines 2157, 2160, and 2165?